query/src/main/java/io/keen/client/java/KeenQueryConstants.java

```diff
@@ -3,10 +3,10 @@
 /**
  * Commonly used constants in the HTTP query request
  *
- * @author claireyoung
+ * @author claireyoung, baumatron, masojus
  * @since 1.0.0, 05/18/15
  */
-public class KeenQueryConstants {
+class KeenQueryConstants {
```

Comment:
Again, we were trying to clean up visibility. Maybe we should plan for a package visibility audit for the next major revision instead?